routes/help.js

```diff
@@ -0,0 +1,38 @@
+const express = require( 'express' )
+const router = express.Router()
+
+const { allHelpItemsQuery } = require( './help/help_response' )
+
+router.get( '/', ( request, response ) => {
+  response.redirect( 'help/howtos' )
+})
+
+router.get( '/howtos', ( request, response ) => {
+  const User = request.app.get('models').User
+  const { user } = request
+
+  User.findById( request.user.id )
+    .then( user => user.getTopics() )
+    .then( helpTopics  => {
```

Comment:
If this fits on a line under 80 characters, please remove braces and make single line.